packages/editor-ui/src/components/ImportParameter.vue

```diff
@@ -0,0 +1,33 @@
+<template>
+	<div class="import-section">
+		<n8n-button
+			type="secondary"
+			:label="$locale.baseText('importParameter.label')"
+			size="mini"
+			@click="onImportCurlClicked"
+		/>
+	</div>
+</template>
+
+<script lang="ts">
+import { IMPORT_CURL } from '@/constants';
+import mixins from 'vue-typed-mixins';
+import { showMessage } from './mixins/showMessage';
+
+export default mixins(showMessage).extend({
+	name: 'ImportParameter',
```

Comment:
Nitpick: We have recently [agreed to use kebab-case for component naming](https://www.notion.so/n8n/Standarizing-naming-aadf11115e9c4272a4f1eff682fa91d1) so this should be `import-parameter`.